index.html

```diff
@@ -49,198 +62,199 @@ <h1>I'm Alier Glad to see you.</h1>
-              <h3>Languages</h3>
-              <img src="images/Union.png" alt="icon" />
-            </div>
-            <div class="tech">
-              <ul>
-                <li class="item1">
-                  <img src="images/Frame9.png" alt="image" />
-                </li>
-                <li class="item2">
-                  <img src="images/Frame10.png" alt="image" />
-                </li>
-                <li class="item3">
-                  <img src="images/Frame11.png" alt="image" />
-                </li>
-              </ul>
-            </div>
-            <div class="frame">
-              <h3>Frameworks</h3>
-              <img src="images/union1.png" alt="icon" />
-            </div>
-            <div class="skill">
-              <h3>Skills</h3>
-              <img src="images/union1.png" alt="icon" />
-            </div>
+      <!--About-me-section begins here-->
+
+      <section class="about-me">
+        <div class="about-text-content-col">
+          <h1 class="about-text">
+            About <br />
+            Myself
+          </h1>
+
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+
+          <p class="lc">LET'S CONNECT</p>
+
+          <ul class="social-icon">
+            <li>
+              <a href=""><img src="images/vector.svg" alt="image" /></a>
+            </li>
+            <li>
+              <a href=""><img src="images/in.svg" alt="image" /></a>
+            </li>
+            <li>
+              <a href=""><img src="images/vectorhand.png" alt="image" /></a>
+            </li>
+            <li>
+              <a href=""><img src="images/vectorhead.png" alt="image" /></a>
+            </li>
+            <li>
+              <a href=""><img src="images/vectorm.png" alt="image" /></a>
+            </li>
+          </ul>
+
+          <div class="resume">
+            <p><a href="">Get my resume</a></p>
+          </div>
+        </div>
+        <div class="lang-col">
+          <div class="lang">
+            <h3>Languages</h3>
+            <img src="images/Union.png" alt="icon" />
+          </div>
+
+          <ul class="tech">
+            <li>
+              <img src="images/javascript.png" alt="image" />
+            </li>
+            <li>
+              <img src="images/html-logo.png" alt="image" />
+            </li>
+            <li>
+              <img src="images/cssLogo.png" alt="image" />
+            </li>
+          </ul>
+
+          <div class="frame">
+            <h3>Frameworks</h3>
+            <img src="images/union1.png" alt="icon" />
+          </div>
+          <div class="skill">
+            <h3>Skills</h3>
+            <img src="images/union1.png" alt="icon" />
           </div>
         </div>
       </section>
```

Comment:
- [ ] Great job but I think you still have to Fix the alignment here so you can use flex/grid to make them aligned beside each other kindly Fix it also the margin and padding I think you need to fix it here to match the Figma design as much as possible

- [ ] Kindly Fix the font weight for the about me header and use `h2` or `h3` as I mentioned above 🙏🏻🙏🏻

- your design 
![aboutd](https://user-images.githubusercontent.com/58553711/178123323-cbeb954d-88df-4597-a3cc-1bf81afe0016.PNG)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/178123343-a6015131-6cfa-4987-a229-ea83cb86d9e1.png)